apps/lib/blockly/es_mx.js

```diff
@@ -200,7 +200,7 @@ Blockly.Msg.LOGIC_TERNARY_IF_FALSE = "si es falso";
 Blockly.Msg.LOGIC_TERNARY_IF_TRUE = "si es verdadero";
 Blockly.Msg.LOGIC_TERNARY_TOOLTIP = "Comprobar la condición en \"probar\". Si la condición es verdadera, devolver el valor \"si es cierto\"; de lo contrario, devolver el valor \"si es falso\".";
 Blockly.Msg.MATH_ADDITION_SYMBOL = "+";
-Blockly.Msg.MATH_ARITHMETIC_HELPURL = "http://es.wikipedia.org/wiki/Aritmética";
+Blockly.Msg.MATH_ARITHMETIC_HELPURL = "http://en.wikipedia.org/wiki/Arithmetic";
```

Comment:
This one is right. The Spanish translation was submitted two years ago and the English translation (the correct one) a year ago. What's interesting is why it updated only now...

https://crowdin.com/translate/codeorg/54/enus-esmx#3352